### PR TITLE
[WIP] Move to MassStorageFile to DiracFile in eos-storage lesson

### PR DESCRIPTION
**Pull request by renaudin**
This is a WIP to update the eos-storage lesson from `MassStorageFile` to `DiracFile` and how to use it.

I still have a few questions as:
- should the `MassStorageFile` part be erased ?
- should I add a part on eos management and how to access files located in `CERN-USER`

What do you think ?

### CODE REVIEW

File: eos-storage.md [hunk 1]
```diff
@@ -13,7 +13,62 @@ minutes: 10
...
 space on a set of hard drives called the [EOS
-service](http://information-technology.web.cern.ch/services/eos-service).
+service](http://information-technology.web.cern.ch/services/eos-service) and
+a grid storage quota of 2TB.
+
+To retrieve a job outputfile, one can use three types of files :  
+- `LocalFile`: the standard one with the output file directly downloaded to
+the `gangadir`.  
+- `DiracFile`: the output file is stored directly on the grid and be accessed
+through XRootD protocol.
```
Comment (ibab) [inline eos-storage.md]: `through the XRootD protocol`

File: eos-storage.md [hunk 2]
```diff
@@ -13,7 +13,62 @@ minutes: 10
 During a real analysis the output of your jobs will quickly grow
 beyond what fits onto your AFS space. CERN provides you with 2TB of
 space on a set of hard drives called the [EOS
-service](http://information-technology.web.cern.ch/services/eos-service).
+service](http://information-technology.web.cern.ch/services/eos-service) and
+a grid storage quota of 2TB.
+
+To retrieve a job outputfile, one can use three types of files :
```
Comment (ibab) [inline eos-storage.md]: The space before `:` can be removed here, and also in a few places below to keep it consistent with the other lessons.

File: eos-storage.md [hunk 3]
```diff
@@ -13,7 +13,62 @@ minutes: 10
...
+
+To add the DiracFile in the configuration of the job we just need :
+```python:
+j = Job(application=DaVinci(version='v41r2'))
+j.backend = Dirac()
+j.name = 'First ganga job'
+j.inputdata = j.application.readInputData('data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py')
+j.application.optsfile = 'code/davinci-grid/ntuple_options_grid.py'
+j.outputfiles = [
+DiracFile('DVntuple.root')
```
Comment (ibab) [inline eos-storage.md]: This line can be indented with 4 spaces.

File: eos-storage.md [hunk 4]
```diff
@@ -13,7 +13,62 @@ minutes: 10
...
+]
+```
+
+When the job is completed, no output is dowloaded but some interesting
+informations are provided by typing `j.outputfiles[0]` :
```
Comment (ibab) [inline eos-storage.md]: `informations` -> information

File: eos-storage.md [hunk 5]
```diff
@@ -13,7 +13,62 @@ minutes: 10
...
+
+```python:
+DiracFile(namePattern='DVntuple.root',
+lfn='/lhcb/user/a/another/2016_11/146255/146255492/DVntuple.root',
```
Comment (ibab) [inline eos-storage.md]: I would indent these two lines to match `namePattern`, or put `namePattern` in the next line with 4 spaces:
```python
DiracFile(namePattern='DVntuple.root',
          lfn='...',
          localDir='...')

File: eos-storage.md [hunk 6]
```diff
@@ -13,7 +13,62 @@ minutes: 10
...
+localDir='/afs/cern.ch/user/a/another/gangadir/workspace/vrenaudi/LocalXML/129/output')
+```
+Apart from the `namePattern` which was set during the configuration of the job,
+we can retrieve the `localDir` which is the path in your gangadir to the output
+of the job and its `lfn` which stands for Logical File Name.
+
+This LFN can be then given as an argument in order to download the file. But
```
Comment (ibab) [inline eos-storage.md]: `can then be given`

**Review by ibab — CHANGES_REQUESTED** [review]
This is really cool and a great addition to the lesson!
I've made some stylistic comments.

File: eos-storage.md (hunk 3 above)
Comment (renaudin) [reply eos-storage.md]: Which line exactly ?

Comment (renaudin) [reply eos-storage.md]: Nervermind, I will indent the DiracFile line

**Comment by ibab** [comment]
Looks good, thank you!

**Review by ibab — APPROVED** [review]

**Pull request merged**